dozer-types/src/arrow_types/to_arrow.rs

```diff
@@ -153,7 +153,7 @@ pub fn map_field_type(typ: FieldType) -> DataType {
         FieldType::Boolean => DataType::Boolean,
         FieldType::String => DataType::Utf8,
         FieldType::Text => DataType::LargeUtf8,
-        FieldType::Decimal => DataType::Decimal256(10, 5), // TODO: Map this correctly
+        FieldType::Decimal => DataType::Decimal256(10, 2), // TODO: Map this correctly
```

Comment:
Would decimals like `199.234` be converted correctly if we make this change?

Reply:
my bad! we are going back to `Decimal256(10, 5)`